test/repl/infra-docker-hoster/docker-compose-destroy.bash

```
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# make sure we have correct cwd
+pushd $(dirname $0) &>/dev/null
+trap 'popd &>/dev/null' EXIT
+
+set -a
+
+. .env
```

Shellcheck (0.11.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)